andela/activo-web
# #159569604 set default centers for new users

**What does this PR do?**

This PR adds functionality to allow a user to set their default center from the profile dropdown if they have never done so or if it's their first time.

**Description of Task to be completed?**

 Given that I am a new user signed up to Activo, I should see a modal window, prompting me to choose 
 a default center from a dropdown list of available centers.

**How should this be manually tested?**

After cloning and running the branch and running the server, if you have not set your default center a modal with a list of centers will appear. Select one center and click continue.

**Any background context you want to provide?**

 At the moment there is no endpoint to try out this feature, therefore the user module is a mock that will contains the current logged in user details. I`m open for feedback.

**What are the relevant pivotal tracker stories?**

[#159569195 As a user I should see a message prompting me to choose a default center if that's my first time on Activo](https://www.pivotaltracker.com/story/show/159569604)

**Screenshots (if appropriate)**

![image](https://user-images.githubusercontent.com/30326444/43965665-883a28ca-9cc8-11e8-97ce-c7f7ecf2481c.png)
![image](https://user-images.githubusercontent.com/30326444/43965682-92e12a8a-9cc8-11e8-9048-023a54d65e0d.png)


**Questions:**

xxx
